test/fixtures/component-interfaces/parse-out.default

```diff
@@ -11,7 +11,7 @@
         "name": "Graphic User\\nInterface"
       },
       {
-        "name": "IMath",
+        "name": "Math",
```

Comment:
I would still expect `name` to be `IMath` but `title` should be `Math`.  

The importatn thing is not to looes information from the diagram, so component name as well as alias should be in the parser result

Reply:
I will implement the title feature with #12